app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    split_command = command.split(" ")
+
+    if (len(split_command) == 3
```

Comment:
raise an error if some of the conditions are not true

Reply:
also, you can create variables for split_command[1] and split_command[2] as you reuse it